src/pkg/backup/details/details.go

```diff
@@ -538,7 +539,7 @@ func (i ExchangeInfo) Headers() []string {
 		return []string{"Contact Name"}
 
 	case ExchangeMail:
-		return []string{"Sender", "Subject", "Received"}
+		return []string{"Sender", "Parent Path", "Subject", "Received"}
```

Comment:
End users will probably respond more naturally to "Folder" than to "path", since they don't think of their mailbox as a directory structure.
```suggestion
		return []string{"Sender", "Folder", "Subject", "Received"}
```

Reply:
Resolved with [8762275](https://github.com/alcionai/corso/pull/2690/commits/876227596f30eb12936b1aecd037ca680acbf0c5)